tests/Interface/core/ImageContainerTest.cpp

```diff
@@ -24,6 +24,45 @@
 #include <buddy/Core/Container.h>
 #include <buddy/DIP/ImageContainer.h>
 
+bool compare_flt(float a,float b)
+{
+	if(a==b) return true;
+        float eps = a-b;
+        if(eps<0)eps=-eps;
+        if(eps<FLT_EPSILON) return true;
```

Comment:
No need for explicit true and false here, you can just return the comparison result itself.